chapter06/mdw/Ex02.pl

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/perl
+
+use 5.010;
+
+my %words;
+
+foreach ( <> ) {
+
+  chomp $_;
+
+  if ( exists $words{$_} ) {
+
+    $words{$_} += 1;
+
+  } else {
```

Comment:
don't think you need this if-else clause, if it doesn't exist it should treat it as 0 and increment that